### PR TITLE
Container build to use conteinerfile

### PR DESCRIPTION
**Pull request by randythomas**

### CODE REVIEW

**Review by rsriniva — APPROVED** [review]

**Comment by rsriniva** [comment]
Please merge this @randythomas

**Comment by jasonwcc** [comment]
Yo guys. The transition is making my class so hard to follow this week. They were looking for Dockerfile, then out of sudden boom Containerfile . Some more some lines differs in those files.
Pls roll out in proper stages. Perhaps in different version. Any notification to trainer about these changes?

**Comment by richardallred** [comment]
> Yo guys. The transition is making my class so hard to follow this week. They were looking for Dockerfile, then out of sudden boom Containerfile . Some more some lines differs in those files.
> Pls roll out in proper stages. Perhaps in different version. Any notification to trainer about these changes?

Jason, 

Sorry if these updates caused issues for you this week.  The DO288 team tried to insulate the 4.5 version of the course (which I _think_ asks students to checkout `master` of this repo) from the 4.6 updates of the course which should ask students to checkout `main`.  I believe `master` should still match the DO288v4.5 course content, but it's possible we overlooked something though.  Are you in a 4.5 or a 4.6 class?  We sent out a notification when we released 4.6 last week, but we definitely could do a better job of communicating w/ instructors in more detail about stuff like this.

We do have a backlog item to improve these "apps" repos to use something more stable like a tag but unfortunately, that change didn't make it into the 4.6 release that went out last week.

Again sorry for the inconvenience, and we appreciate your feedback.
